images/phpstorm/Dockerfile.ubuntu

```diff
@@ -0,0 +1,23 @@
+FROM codercom/enterprise-multieditor:ubuntu
+
+# Run everything as root
+USER root
+
+# Install php interpreter and CGI
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND="noninteractive" apt-get install --yes \
+      php \
+      php7.4-cgi \
```

Comment:
**thought**: It might good for us to create a php base to extend from, since this version is hard-coded here